src/interp/interp-wasi.cc

```diff
@@ -66,8 +76,49 @@ class WasiInstance {
     return Result::Ok;
   }
 
+  Result fd_fdstat_get(const Values& params, Values& results, Trap::Ptr* trap) {
```

Comment:
might be nice to duplicate the WASI function signature here, with the WASI type names

Reply:
You mean in comments?   I added links at the top of the docs and headers for the wasi API.